### ISSUE TITLE
psalm 4.4.0 gives undefined variable errors inside psalm code

### ISSUE THREAD

**Issue by thomasbley** [vimeo/psalm#5010]
Psalm 4.4.0 gives these errors when running, 4.3.2 and 4.3.1 did not give these errors:

```
PHP Notice:  Undefined variable: result in phar:///usr/local/sbin/psalm_4400.phar/src/Psalm/Internal/ExecutionEnvironment/SystemCommandExecutor.php on line 25
Notice: Undefined variable: result in phar:///usr/local/sbin/psalm_4400.phar/src/Psalm/Internal/ExecutionEnvironment/SystemCommandExecutor.php on line 25
PHP Notice:  Undefined variable: returnValue in phar:///usr/local/sbin/psalm_4400.phar/src/Psalm/Internal/ExecutionEnvironment/SystemCommandExecutor.php on line 25
Notice: Undefined variable: returnValue in phar:///usr/local/sbin/psalm_4400.phar/src/Psalm/Internal/ExecutionEnvironment/SystemCommandExecutor.php on line 25
PHP Warning:  exec() has been disabled for security reasons in phar:///usr/local/sbin/psalm_4400.phar/src/Psalm/Internal/ExecutionEnvironment/SystemCommandExecutor.php on line 25
Warning: exec() has been disabled for security reasons in phar:///usr/local/sbin/psalm_4400.phar/src/Psalm/Internal/ExecutionEnvironment/SystemCommandExecutor.php on line 25
PHP Notice:  Undefined variable: returnValue in phar:///usr/local/sbin/psalm_4400.phar/src/Psalm/Internal/ExecutionEnvironment/SystemCommandExecutor.php on line 26
Notice: Undefined variable: returnValue in phar:///usr/local/sbin/psalm_4400.phar/src/Psalm/Internal/ExecutionEnvironment/SystemCommandExecutor.php on line 26
PHP Notice:  Undefined variable: returnValue in phar:///usr/local/sbin/psalm_4400.phar/src/Psalm/Internal/ExecutionEnvironment/SystemCommandExecutor.php on line 30
Notice: Undefined variable: returnValue in phar:///usr/local/sbin/psalm_4400.phar/src/Psalm/Internal/ExecutionEnvironment/SystemCommandExecutor.php on line 30
```

I'm using PHP 7.4.13 with disable_functions = "shell_exec,proc_open,passthru,highlight_file,system,popen,exec" in php.ini.

**Comment by thomasbley** [comment]
The code is trying to execute "git branch" from GitInfoCollector.php.

#0 src/Psalm/Internal/ExecutionEnvironment/GitInfoCollector.php(58): Psalm\Internal\ExecutionEnvironment\SystemCommandExecutor->execute()
#1 src/Psalm/Internal/ExecutionEnvironment/GitInfoCollector.php(44): Psalm\Internal\ExecutionEnvironment\GitInfoCollector->collectBranch()
#2 src/Psalm/IssueBuffer.php(548): Psalm\Internal\ExecutionEnvironment\GitInfoCollector->collect()
#3 src/psalm.php(807): Psalm\IssueBuffer::finish()

**Comment by muglug** [comment]
Thanks! Psalm 4.4.1 is now released with a fix!
